src/events.ts

```diff
@@ -36,7 +37,7 @@ export class EventDispatcher {
     this.listeners[type]?.delete(listener);
   }
```

Comment:
Why aren't we passing `<T>` to `once`, `has`, and `off`?

Reply:
Added to `once`, can't think of any type safety added by the others can you?

Reply:
No, `once` make sense.